src/bootstrap/src/core/build_steps/gcc.rs

```diff
@@ -41,7 +41,8 @@ pub fn prebuilt_gcc_config(builder: &Builder<'_>, target: TargetSelection) -> Gc
     // FIXME (GuillaumeGomez): To be done once gccjit has been built in the CI.
     // builder.config.maybe_download_ci_gcc();
 
-    let root = "src/gcc";
+    let root =
```

Comment:
Ideally this should be `builder.src.join("src/gcc")`, as the current working directory might not always be the source root (although that's rare).

Reply:
Much better indeed! :+1: